js/plot/util/annotations/annotationdefinitions.js

```diff
@@ -34,6 +34,27 @@ var definitions = {
       var b = '\'s threshold for reporting a numerical value. Your actual BG value was higher or lower than displayed.';
       return defs.stitch(a, b, source);
     },
+    'animas/bolus/extended-equal-split': function(source, defs) {
```

Comment:
I looked into why these aren't showing up when embedded in wizards. The filtering is fine, the problem is that we call a method to draw annotations for wizards and then we call a method to draw annotations for non-wizard boluses, and the latter overwrite the former. Not going down the rabbithole to try to tease that apart right now 😕